### DISCUSSION TITLE
Welcome to RulesEngine Discussions!

### DISCUSSION THREAD

**Discussion by abbasc52** [microsoft/RulesEngine#74] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mahen83** [comment]
localParams is cool feature.. having it at WorkFlow level would be really helpful. Hope you consider to bring in this change. Thanks

**Reply by abbasc52** [reply]
@mahen83 it is available as a part of v3.1.0
